thirdeye/thirdeye-hadoop/pom.xml

```diff
@@ -45,7 +45,6 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
-            <scope>provided</scope>
```

Comment:
I think a better approach would be to keep Hadoop a provided dependency, but to NOT shade org.apache.hadoop classes in the shaded jar. See https://maven.apache.org/plugins/maven-shade-plugin/examples/class-relocation.html for details.